appveyor.yml

```diff
@@ -27,6 +27,11 @@
               BOND_VS_NUM: 14
               BOND_ARCH: 32
               BOND_BOOST: 60
+            - BOND_BUILD: Python
+              BOND_VS: "Visual Studio 14 2015"
+              BOND_VS_NUM: 14
+              BOND_ARCH: 32
```

Comment:
We've got a 32-bit variant already. Do you want to use 64 here for increased coverage?